### DISCUSSION TITLE
Error access /api/v1/enroll

### DISCUSSION THREAD

**Discussion by douglasjava** [hyperledger/fabric-gateway#624] [Q&A]
Hello everyone, 

when the sdk will execute the call with method httpPost class HFCAClient
This error is happening **java.security.InvalidAlgorithmParameterException: the trustAnchors parameter must be non-empty**

this method String httpPost(String url, String body, UsernamePasswordCredentials credentials) throws Exception {

HttpResponse response = client.execute(httpPost, context);

  ```
<dependency>
    <groupId>org.hyperledger.fabric</groupId>
    <artifactId>fabric-gateway-java</artifactId>
    <version>2.2.8</version>
  </dependency>
```

**Comment by bestbeforetoday** [comment]
This question does not relate to the Fabric Gateway client API.
